CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.2.0
+
+This is a major Teleport release with a focus on new features and bug fixes.
```

Comment:
Should we standardise our language to match SemVar https://semver.org/  in this case, this will be a Minor Teleport Release.

Reply:
Done. f62c4620dd565a3894a900ab435f11a991e5e9e6 uses minor.